### PR TITLE
Addig new resources

### PR DESCRIPTION
**Pull request by PaolaDaft**
Hi✨, I added 4 new resources according to issue #5, in my commits I explain the resources that I added

### CODE REVIEW

File: src/assets/data/resources.json [hunk 1]
```diff
@@ -175,5 +175,33 @@
...
+    {
+        "title": "Pro Git",
+        "image": "https://git-scm.com/images/progit2.png",
+        "description": "The guide to Git and its usage.",
+        "url": "https://git-scm.com/book/en/v2",
+        "tags": ["git", "books", "documentation", "developer"]
+    },
+    {
+        "title": "Eloquent JavaScript",
+        "image": "https://nostarch.com/sites/default/files/styles/uc_product_full/public/EJS3_front.png?itok=2qHtZpJa",
```
Comment (TenTraicion) [inline src/assets/data/resources.json:188]: Kindly change the image link with a proper github hosted .png image link. Thanks.

File: src/assets/data/resources.json [hunk 2]
```diff
@@ -175,5 +175,33 @@
...
+        "description": "This is a book about JavaScript, programming, and the wonders of the digital.",
+        "url": "https://eloquentjavascript.net/",
+        "tags": ["JavaScript", "books", "developer"]
+    },
+    {
+        "title": "GoHugo",
+        "image": "https://d33wubrfki0l68.cloudfront.net/c38c7334cc3f23585738e40334284fddcaf03d5e/2e17c/images/hugo-logo-wide.svg",
```
Comment (TenTraicion) [inline src/assets/data/resources.json:195]: Kindly change the image link with a proper github hosted .png image link. Thanks.

File: src/assets/data/resources.json [hunk 3]
```diff
@@ -175,5 +175,33 @@
...
+        "description": "GoHugo is a tool to create static websites (js, html and css). It helps you to build simple websites, the most common uses are: blogs, landing pages, documentation, etc.",
+        "url": "https://gohugo.io/",
+        "tags": ["markdown", "tools", "blog"]
+    },
+    {
+        "title": "How to Contribute to an Open Source Project on GitHub",
+        "image": "https://egghead.io/_next/image?url=https%3A%2F%2Fd2eip9sf3oo6c2.cloudfront.net%2Fplaylists%2Fsquare_covers%2F000%2F432%2F579%2Fsquare_480%2FEGH_JSopensource_final.png&w=1920&q=75",
```
Comment (TenTraicion) [inline src/assets/data/resources.json:202]: Kindly change the image link with a proper github hosted .png image link. Thanks.

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@PaolaDaft, Thanks for your contribution. Kindly change your image links with a github hosted image URLs. You can just paste or drag your image in any comment/discussion/issue in github and you will get a link, just paste it here. Thanks.

File: src/assets/data/resources.json (hunk 1 above)
Comment (PaolaDaft) [reply src/assets/data/resources.json:188]: EJS3_front
1_SwbEdd_9O-VkpkqeTS72VA
how-to-contribute-to-an-open-source-project-on-github

File: src/assets/data/resources.json (hunk 2 above)
Comment (PaolaDaft) [reply src/assets/data/resources.json]: Done✨

File: src/assets/data/resources.json (hunk 3 above)
Comment (PaolaDaft) [reply src/assets/data/resources.json]: Done✨

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@PaolaDaft, Congrats on your PR-Merge. and Thanks for the contribution.

**Pull request merged**